jib-core/src/integration-test/java/com/google/cloud/tools/jib/builder/BuildStepsIntegrationTest.java

```diff
@@ -165,23 +179,35 @@ public void testSteps_forBuildToDockerRegistry()
     BuildResult image1 =
         BuildSteps.forBuildToDockerRegistry(
                 getBuildConfigurationBuilder(
-                        ImageReference.of("gcr.io", "distroless/java", "latest"),
+                        ImageReference.of("gcr.io", "distroless/java", DISTROLESS_DIGEST),
```

Comment:
Huh.  Was there a reason for this change?  Does it speed up the tests?

Reply:
Since this test depends on the number of layers in the base image, we decided to use a digest instead of `latest` to avoid flakiness.

Reply:
👍 let's hope gcr.io/distroless doesn't do GCs :-)